.eslintignore

```diff
@@ -3,3 +3,4 @@ node_modules
 coverage
 build
 **/aggregated-translations/*.*
+**/react-datepicker/*.*
```

Comment:
Can we make this a direct path ?
```
packages/terra-datapicker/src/react-data-picker/**
```

Reply:
https://github.com/cerner/terra-framework/pull/573/commits/07793bd62b5b0afc00c050a9c5f83cba38745cbf